test/index.test.js

```diff
@@ -381,6 +381,95 @@ describe('index', function () {
             });
         });
 
+        it('creates containers from a private docker registry and starts them', () => {
+            container = 'docker-registry.foo.bar:1111/someImage:latest';
+
+            const launcherContainer = {
```

Comment:
Because of the way the test suite is constructed, there are other test cases that have a different value for `launcherContainer`. This, in turn, affects how the mocks react to certain input.

I've gone ahead and refactored the tests enough to consolidate some shared values.